docker/hq-compose.yml

```diff
@@ -100,14 +104,15 @@ services:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.4.13/data
 
   kafka:
-      image: wurstmeister/kafka:0.8.2.2-1
+      image: wurstmeister/kafka:2.13-2.6.0
```

Comment:
Is this related to the formplayer revamp?

Reply:
It was not - I did initially have some trouble getting kafka to work with commcare running in docker but I think our config was just wrong. I can revert this change though this is now the same version we're using on prod.

Reply:
Fine to keep it, assuming I'm understanding correctly that it's basically cleanup.